src/System.Security.Cryptography.X509Certificates/tests/ChainTests.cs

```diff
@@ -260,6 +261,70 @@ public static void BuildChain_WithCertificatePolicy_NoMatch()
         }
 
         [Fact]
+        [OuterLoop(/* Modifies user certificate store */)]
+        public static void BuildChain_WithRootCertInUserAndRootCertStores()
+        {
+            // Verifies that if a root cert is placed in both a root and 
+
+            using (var microsoftDotCom = new X509Certificate2(TestData.MicrosoftDotComSslCertBytes))
+            using (var microsoftDotComRoot = new X509Certificate2(TestData.MicrosoftDotComRootBytes))
+            using (var machineRootStore = new X509Store(StoreName.Root, StoreLocation.LocalMachine))
+            using (var userRootStore = new X509Store(StoreName.Root, StoreLocation.CurrentUser))
+            {
+                try
+                {
+                    machineRootStore.Open(OpenFlags.ReadOnly);
+                    userRootStore.Open(OpenFlags.ReadWrite);
+
+                    // Check that the certificate is installed on the local machine
+                    var rootCerts = machineRootStore.Certificates.Find(X509FindType.FindByThumbprint, microsoftDotComRoot.Thumbprint, false);
```

Comment:
Thanks - I was just using CI to verify sanity (this was a straight-port from a test program I was using); the changes are being made now to avoid using .Find